examples/ruby/lib/twilio-ruby/rest/api/v2010/account/call.rb

```diff
@@ -130,8 +130,6 @@ def feedback_call_summary(sid = :unset)
               unless @feedback_call_summary
                 @feedback_call_summary = FeedbackCallSummaryList.new(
                   @version,
-                  account_sid: @solution[:account_sid],
-                  call_test_integer: @solution[:test_integer]
```

Comment:
same question as above, understanding was this would get renamed and not removed.

Reply:
Earlier fix was statically adding these variables with `className_paramName`, since that's removed and dependents details are used now, this got removed.
I will update the spec file to include dependents info and add a tc, as suggested in earlier comments.